frontend/test/metabase/scenarios/dashboard-filters/reproductions/15279-gracefully-deal-with-corrupted-filter.cy.spec.js

```diff
@@ -38,7 +38,7 @@ describe("issue 15279", () => {
     cy.signInAsAdmin();
   });
 
-  it("filters should work even if one of them is corrupted (metabase #15279)", () => {
+  it("a corrupted parameter filter should still appear in the UI (metabase #15279)", () => {
```

Comment:
the new query endpoint results in a 500 error because these corrupted filters don't match the parameter schema (non empty string `name` prop). since this endpoint depends on what parameters are defined on a dashboard, I think this is likely unavoidable.